buckaroo/GitManager.fs

```diff
@@ -118,10 +115,10 @@ type GitManager (git : IGit, cacheDirectory : string) =
       refsCache <- refsCache |> Map.add url refs
       return refs
   }
-  member this.FetchFile (url : string) (revision : Revision) (file : string) (hint : Option<Branch>) : Async<string> =
+  member this.FetchFile (url : string) (revision : Revision) (file : string) : Async<string> =
     async {
       let! targetDirectory = this.Clone(url)
-      do! this.FetchCommit url revision hint
+      do! this.FetchCommit url revision
```

Comment:
Hint is gone :tada: but is this now slow?

Reply:
Now it will always query the defaultBranch. 
This is a good guess that will lead to quick results in most cases.
In future we can extract more information from the VersionSets